src/components/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -59,15 +63,18 @@
         <DatePicker
           v-if="hasCalendar && showCalendar"
           v-model="calendarVModel"
+          borderless
+          color="blue"
```

Comment:
What is `color="blue"` ?

Reply:
https://vcalendar.io/calendar/theme.html#colors

I can test the removal of this prop in the sandbox. My guess is that it's acting as a safe fallback, since `blue` seesm to be the closest VCalendar color theme to what we have in Konnect.

Reply:
works for me, just wanted to make sure it wasn't a 1:1 color